app/src/main/res/layout/fragment_session_detail.xml

```diff
@@ -81,6 +81,7 @@
                         android:layout_width="match_parent"
                         android:layout_height="?attr/actionBarSize"
                         android:background="?attr/selectableItemBackground"
+                        style="@style/BaseToolbar"
                         app:contentInsetStartWithNavigation="0dp"
```

Comment:
If you use `@style/BaseToolbar`, you can remove `app:contentInsetStartWithNavigation="0dp"`.